src/runtime_src/core/common/info_memory.cpp

```diff
@@ -334,12 +334,18 @@ struct memory_info_collector
     : device(dev)
     , mem_topo_raw(xrt_core::device_query<xq::mem_topology_raw>(device))
     , grp_topo_raw(xrt_core::device_query<xq::group_topology>(device))
-    , mem_temp_raw(xrt_core::device_query<xq::temp_by_mem_topology>(device))
     , mem_topo(mem_topo_raw.empty() ? nullptr : reinterpret_cast<const mem_topology*>(mem_topo_raw.data()))
     , grp_topo(grp_topo_raw.empty() ? nullptr : reinterpret_cast<const mem_topology*>(grp_topo_raw.data()))
     , mem_stat(xrt_core::device_query<xq::memstat_raw>(device))
-    , mem_temp(mem_temp_raw.empty() ? nullptr : reinterpret_cast<const uint32_t*>(mem_temp_raw.data()))
+    , mem_temp(0)
```

Comment:
`nullptr` plutôt?

Reply:
Hi @keryell, thanks for the comments. I do see your point, but since it is a cherry-pick, I don't want to make changes to this PR.